packages/conf-glpk/conf-glpk.1/opam

```diff
@@ -6,7 +6,9 @@ authors: ["Andrew Makhorin" "GNU"]
 license: "GPL-3.0-or-later"
 build: [
   ["sh" "-c" "echo '#include <glpk.h>' > test.c"]
-  ["cc" "-E" "test.c"]
+  ["cc" "-E" "test.c"] {os = "linux"}
+  ["cc" "-E" "-I/usr/local/include" "test.c"] {os = "freebsd"}
+  ["cc" "-E" "-I/usr/local/include" "-I/opt/homebrew/include" "test.c"] {os = "macos" & os-distribution = "homebrew"}
```

Comment:
This will still fail on all nonstandard homebrew installations, but it is generally a problem of many packages